<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>874</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BEST Gallery - 3.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Logo_apli.png</normaloff>Logo_apli.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1131</width>
      <height>841</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Descargar</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>641</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>641</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bdescarga">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>620</y>
        <width>311</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Descargar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="try_conexion_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>640</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reestablecer conexión</string>
      </property>
     </widget>
     <widget class="QLabel" name="etiqueta2_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>641</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_archivo_elegido">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>801</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="logo_best">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>10</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="babrir_carpeta">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>630</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir carpeta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="volver_atras">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver atrás</string>
      </property>
     </widget>
     <widget class="QPushButton" name="brecilaje">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>510</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_carpeta">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>180</y>
        <width>180</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bnueva_carpeta">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>210</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Nueva Carpeta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_cambiarnom">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>290</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Renombrar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Subir</string>
     </attribute>
     <widget class="QPushButton" name="archivos">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>660</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="text">
       <string>Examinar</string>
      </property>
      <property name="autoRepeatInterval">
       <number>100</number>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>660</y>
        <width>331</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="try_conexion">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>680</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reestablecer conexión</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>461</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Elige el archivo con extensión .PNG que quieras subir a BEST Gallery.</string>
      </property>
     </widget>
     <widget class="QLabel" name="etiqueta2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>641</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="etiqueta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>681</width>
        <height>151</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="prefoto">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>721</width>
        <height>411</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="logo_best2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>40</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>641</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>540</y>
        <width>341</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Recuerda que debes seleccionar la carpeta en la 
que quieres subir el archivo  en la pestaña &quot;Descargar&quot;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Buscar...</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_filtro">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>561</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Buscar_filtro">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>110</y>
        <width>191</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lista_filtro">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>581</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="ir_dir_filtro">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>690</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Situarse en el directorio/en su directorio</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>641</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSubir">
   <property name="text">
    <string>Subir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
